_posts/en/newsletters/2025-05-02-newsletter.md

```diff
@@ -0,0 +1,86 @@
+---
+title: 'Bitcoin Optech Newsletter #352'
+permalink: /en/newsletters/2025/05/02/
+name: 2025-05-02-newsletter
+slug: 2025-05-02-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Comparison of cluster linearization techniques:**
+  Pieter Wuille [posted][wuille clustrade] about some of the fundamental
+  tradeoffs between three different cluster linearization techniques,
+  following up with [benchmarks][wuille clusbench] of implementations of
+  each.  Several other developers discussed the results and asked
+  clarifying questions, to which Wuille responded.
+
+- **Increasing or removing Bitcoin Core's `OP_RETURN` size limit:**
+  in a thread on Bitcoin-Dev, several developers discussed changing or
+  removing Bitcoin Core's default limit for `OP_RETURN` data carrier
+  outputs.  A subsequent Bitcoin Core [pull request][bitcoin core
+  #32359] saw additional discussion.  Rather than attempt to summarize
+  the entire voluminous discussion, we'll summarize only what we
+  thought was the most compelling argument against the change and for
```

Comment:
```suggestion
  thought was the most compelling argument for and against
```
And maybe also reverse the order of the two bullet points below. I just think "for and against" is something people are used to saying and hearing, rather than "against and for".